### PR TITLE
Added alumni page

### PR DESCRIPTION
**Pull request by sourishg**
Batch of 2014 added.

### CODE REVIEW

File: _includes/alumni.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!-- Begin Team List-->
+<section id="team" class="team">
+    <div class="content-wrapper">
+        {% for hierarchy in site.data.alumni %}
```
Comment (narayanaditya95) [inline _includes/alumni.html]: Change it to year

**Comment by narayanaditya95** [comment]
Add alumni page to the header as well.

File: _data/team.yml [hunk 2]
```diff
@@ -209,3 +209,96 @@
       team: true
       social:
         - {name: "facebook", link: "https://www.facebook.com/sakshi.agarwal.5682944?fref=ts"}
+- title: "Sub Head"
+  team:
+    -
+      id: 25
```
Comment (narayanaditya95) [inline _data/team.yml]: Alphabetically

**Comment by narayanaditya95** [comment]
Convert images to 250px*250px. Run them through the lossy script

**Comment by sourishg** [comment]
Redirects are added.

**Comment by narayanaditya95** [comment]
Why you make it tough for me to review 3 PRs merged into one :p

**Comment by sourishg** [comment]
I was just pushing my commits into master. You should merge the PRs quickly
I guess. :p

**Comment by sourishg** [comment]
There is an issue. The text inside the contact cards in by default white, so cards with thumbnails are looking really bad. Since I don't completely understand how the css works here, I suggest you fix this issue.

**Comment by narayanaditya95** [comment]
Push it to your gh-pages branch so that I can see whats wrong.

**Comment by sourishg** [comment]
I pushed it. There seems to be a problem with the base-url. Should I change it?

**Comment by narayanaditya95** [comment]
- Some photographs that aren't square are rendered by tile. This happens for Avinash, Nilanjana and many others. Resize and crop them to 250px*250px.
- Use a [generic profile pic](http://walyou.com/wp-content/uploads//2010/12/facebook-profile-picture-no-pic-avatar.jpg) for alumnus we dont have a pic for. Again crop and resize to 250px*250px.

Hope it solves the problem.

**Comment by narayanaditya95** [comment]
@sourishg This PR is going stale! Finish the PR by today!

**Comment by sourishg** [comment]
@narayanaditya95 I asked Rajat and Shweta to resize the images. They didn't send me the images yet. I will add the generic picture though.

**Pull request merged**